developer-guideline.md

```diff
@@ -0,0 +1,163 @@
+## Dapp Browser Developer Guideline
+
+#### This guideline is to help developers develop Dapps that can be used in rWallet's Dapp Browser.
+
+#### 1. Basic
+
+##### Unit
+
+"wei" are the smallest chain unit, and you should always make calculations in wei and convert only for display reasons.
+
+```
+web3.utils.toWei('1');
+> "1000000000000000000"
+```
+
+##### Transaction
+Every transaction requires a certain gas fee. The total gas fee = gas * gasPrice.
+
+If you want to transfer 1 RBTC to your friend, and the gas is 500000 wei, gasPrice is 20 gwei (1 gwei = 1000000000 wei), so the total gas fee is 1e16 wei (is equal to 0.01 RBTC) , you need to cost 1.01 RBTC, then your friend will get 1 RBTC.
+
+The send transaction steps:
+
+1. call getTransactionFees method to calculate the transaction fee
+2. generate transaction object: { symbol, type, sender, receiver, value, memo, gas, gasPrice }
```

Comment:
Is step 2 duplicate of step 3?

Reply:
step 2's transaction object does not contain the 'nonce' and 'chainId' field, but step 3's transaction object contains.

Reply:
Okay I see.